index.js

```diff
@@ -0,0 +1,23 @@
+//*  2020 - 03 - 19 //
+// 03 - 19 - 2020 //
+
+let oldDate = "2020 - 03 - 19";
+
+const date = oldDate.split(" ");
+
+let temp = date[0];
+date[0] = date[4];
+date[4] = temp;
+
+let temp2 = date [0];
+date[0] = date[2];
+date[2] = temp2; 
+
+date[1] = "/";
```

Comment:
Is there any other way of adding `/` without explicitly doing so? 
Take a look at string replacement.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace